cmd/bee/cmd/db.go

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build oldlocalstore
```

Comment:
Some comment would be nice to have about this build tag.

Reply:
As I mentioned, there is no information here as this is temporary, just so that we can build the binary.